src/ingest_validation_tools/plugin_validator.py

```diff
@@ -155,6 +161,6 @@ def validation_error_iter(
     if not base_dir.is_dir():
         raise ValidatorError(f"{base_dir} should be the base directory of a dataset")
     for cls in validation_class_iter(plugin_dir):
-        validator = cls(base_dir, assay_type)
+        validator = cls(base_dir, assay_type, contains)
```

Comment:
As long as none of the existing Plugins override the __init__ class defined on line 46 then this should be fine. Just making sure that that is the case.

Reply:
Good point, I will double check that.

Reply:
All good with this at present.